webapp/mypackage/.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
```

Comment:
looks you still have __pycache__ files being committed. 
You may need to remove them from version control. You can stop tracking a file with
`git rm --cached <filename>`

Reply:
used git rm -r -cached __pycache__